src/proxy/collection.rs

```diff
@@ -10,7 +10,7 @@
 use serde::{Deserialize, Serialize};
 use std::collections::HashMap;
 use zbus::{
-    dbus_proxy,
+    proxy,
```

Comment:
Nit: Let's remove this import and qualify it instead for consistency with the other files.